benchmarks/torchbench_model.py

```diff
@@ -56,13 +56,28 @@
     "timm_vovnet",
     "vgg16",
     "hf_T5",
-    # PyTorch/benchmark sets its optimizer as SGD.
-    # Otherwise, OOMs.
-    "llama_v2_7b_16h",
 }
 
 # Skip the experiment of a model if any of the experiment configs in the list is fully matched
 DENY_LIST = {
+    "cm3leon_generate": [
+        {
+            "test": "train",
+        },
+        {
+            "test": "eval",
+            "xla": "PJRT",
+        },
+    ],  # no install.py
+    "hf_T5_generate": [
+        {
+            "test": "train",
+        },
+        {
+            "test": "eval",
+            "xla": "PJRT",
+        },
+    ],  # no install.py
```

Comment:
Given your definition for "strict-compatible", I think `{"test": "eval", "xla": "PJRT"}` should be there, shouldn't it? It's not that they fail, but they do timeout.

Reply:
Just to make it explicit: with this setting, we are also skipping `dynamo+openxla` backend. While that doesn't timeout, it does run into the [`cachedComputation` check error](https://github.com/pytorch/xla/issues/5967).

Reply:
Thanks for the info!

Reply:
Right, they do timeout, but we want to also kick out the timeout test when we do test automation to prevent job failure.